src/uct/tcp/tcp.h

```diff
@@ -17,7 +17,7 @@
 
 #define UCT_TCP_NAME                "tcp"
 
-/* How many events to wait for in epoll_wait */
+/* Maximum number of times to poll on a ready socket. */
```

Comment:
It sounds like a wrong statement
It should be something:
`Maximum number of events to wait from event set`

Reply:
fixed